marklogic/marklogic-data-hub
# DHFPROD-8859: Splitting Ui config to multiple config files

### Description

To test:

1. Build the Twizzlers app and Entity Viewer project using the PR branch. 
2. Deploy and load modules and test that it works as expected.
3. Update the configuration somehow, which is now in separate *.config.sjs files. Change the data display, an icon, a color, whatever. Config files are here:
https://github.com/rahulvudutala/marklogic-data-hub/tree/DHFPROD-8859/examples/entity-viewer-v2/entity-viewer-client/src/main/ml-modules/root/explore-data/ui-config
4. Load the updated config modules.
5. Test that the change shows up in the app.

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:
